### PR TITLE
rossanodan-1

### LINKED ISSUE #1: Add contributing rules
The project needs clear contributing rules so that people can follow them contributing to the codebase.
Is there any template we can customise and use?

### PR DESCRIPTION
**Pull request by rossanodan**
Improved the README and the CONTRIBUTING guide.

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Maybe we want to include `commit` structure too using something like http://conventionalcommits.org

File: README.md [hunk 1]
```diff
@@ -2,42 +2,68 @@
 
 EddieBotUI is the front end application that shows Eddie Joaoude Discord Community's public stats.
```
Comment (eddiejaoude) [inline README.md]: Typo. Maybe we should always put this in bold?

```suggestion
EddieBotUI is the front end application that shows Eddie Jaoude Discord Community's public stats.
```

With bold

```suggestion
EddieBotUI is the front end application that shows **Eddie Jaoude** Discord Community's public stats.
```

**Comment by rossanodan** [comment]
> Maybe we want to include `commit` structure too using something like http://conventionalcommits.org

That's a good idea, it helps with consistency 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+If you have an idea for a new feature, please start by [searching the issues](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues) to check that the feature has not already been suggested and then suggest it by [opening a new issue](https://github.com/EddieJaoudeCommunity/EddieBotUI/issues/new), as adding new features to EddieBotUI first requires some analysis around the design and spec.
+
+## Other ways to help
+
+If you're not a developer there are still plenty of ways that you can help. We always need help with:
+
+- Helping our EddieJaoudeCommunity on the [chat](https://discord.com/invite/jZQs6Wu)
+- Improving the [documentation](https://github.com/EddieJaoudeCommunity/EddieBotUI/tree/develop/docs) and writing tutorials
+- Testing and quality assurance
+- Promoting EddieJaoudeCommunity to others by blogging, vlogging, code streaming, talking etc.
```
Comment (eddiejaoude) [inline CONTRIBUTING.md]: Here it is `EddieJaoudeCommunity` and other places it is 3 words. Not sure which is better. I think easier to read at 3 separate words in bold?

**Comment by eddiejaoude** [comment]
Looks great 👍 . I left 1 small comment about consistency.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rossanodan) [reply CONTRIBUTING.md]: I'll fix that ;)

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**